lua/omnisharp_extended.lua

```diff
@@ -280,28 +272,28 @@ M.handler_telescope = function(err, result, ctx, config, opts)
   M.handle_locations_telescope(locations)
 end
 
-M.telescope_lsp_definitions = function(opts)
+M.telescope_lsp_definitions = function()
```

Comment:
opts passing shouldnt be removed from telescope path, as it was there to allow changing telescope prompts/config. It seems I originally forgot to pass it to `handle_locations_telescope`.